examples/proc_FIR.py

```diff
@@ -15,16 +15,15 @@
 T1_list = []
 clock_correction = True
 plot_fit = True
-# I'm using the color cycle for more plots than just those made by
-# rough_table_of_integrals, I don't have this iterating correctly though
-prop_cycle = plt.rcParams["axes.prop_cycle"]
-color_cycle = prop_cycle.by_key()["color"]
-thiscolor = next(iter(color_cycle))
-
-# should I be using something like this instead?
-# colorcyc_list = plt.rcParams["axes.prop_cycle"].by_key()["color"]
-# colorcyc = cycle(colorcyc_list)
-
+# TODO ☐: to directly answer your question, the thing you get from
+#         matplotlib isn't a cycle, but a "cycler", which is different.
+#         You need to feed it to "cycle" from "itertools" in order to
+#         make it into a "cycle" and be able to use "next" on it.
+#         BUT you should't do that, anyways.  The nddata object that
+#         comes out of rough_table_of_integrals will always plot with
+#         the same color (as will copies of it).  You can also get the
+#         color of that object (e.g. for use in an unrelated dataset
+#         axvline, etc) with `s.get_plot_color()`
```

Comment:
do read this explanation -- the long story short is that because of this, I'm able to just delete several keyword arguments below, and everything works.